docker/brew.Dockerfile

```diff
@@ -128,6 +128,10 @@ RUN $YARN install --frozen-lockfile --production --network-timeout 600000
 FROM registry.redhat.io/ubi9/nodejs-18-minimal:1 AS runner
 USER 0
 
+RUN microdnf update -y && \
+  microdnf install -y python3 python3-pip && \
+  microdnf clean all
```

Comment:
Do you also need this, as per the  upstream Dockerfile??

```
  pip3 install mkdocs-techdocs-core==1.* && \
```

Reply:
@christophe-f could answer I assume. It mainly depends on how techdocs is supported on downstream...
I don't have sufficient context and history to answer TBF.

Reply:
at least it seems that installing `mkdocs-techdocs-core` also installs the `yaml` package which we need in the dynamic-plugins-install.py script.